CHANGELOG.md

```diff
@@ -5,6 +5,18 @@ project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+### Runtime
+
+- ast: Annotations scoped to package carries across module ([#5251](https://github.com/open-policy-agent/opa/issues/5251)).
+
+### Breaking changes
+
+The change to the `package` scope in [#5251](https://github.com/open-policy-agent/opa/issues/5251) may cause compile-time errors and behavioural changes to type checking when the `schemas` annotation is used, and to rules calling the `rego.metadata.chain()` built-in function:
+
+- Existing projects with the same package declared in multiple files will trigger a `rego_type_error: package annotation redeclared` error _if_ two ore more of these are annotated with the `package` scope.
```

Comment:
ore -> or

Reply:
As long as there are no conflicting attributes, could we try and merge the annotations from both files rather than failing?

Reply:
A simple merge that failed on any annotation conflict should be fine, I think. If we make that change, we should do the same for the `document` scope; so let's create a separate ticket for that.